src/components/transactions/message-view/WithdrawDelegationRewardMessageDetails.vue

```diff
@@ -58,9 +107,56 @@ export default {
     }
   },
   computed: {
-    validator() {
-      return this.validators[this.transaction.value.validator_address] || false
+    getValidators() {
+      if (this.transaction.value.length > 1) {
+        let validators = []
+        this.transaction.value.forEach(msg => {
+          validators.push(this.validators[msg.value.validator_address] || {})
+        })
+        return validators
+      }
+      // hack for something I don't fully understand yet. a [{...}, [Observer]] structure coming up in Terra
+      if (this.transaction.value.validator_address) {
+        return [this.validators[this.transaction.value.validator_address]] || []
+      } else {
+        return [this.transaction.value[0].value.validator_address] || []
+      }
     }
   }
 }
 </script>
+<style scoped>
+.validators-images-row {
+  display: flex;
+  flex-direction: row;
+}
+.multi-claim-reward-row {
+  display: flex;
+}
+.multi-claim-reward-h3 {
+  margin-right: 20px;
+}
+.row-validator-image {
+  float: left;
+  padding: 5px;
+}
+.validator-image {
+  border-radius: 100%;
+  padding: -1px;
+  height: 1rem;
+  width: 1rem;
+  vertical-align: middle;
```

Comment:
this probably isn't necessary. if you're trying to have them centered in the row (which they currently aren't) you can use flex styles like `align-items: center`